From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Guido Urdaneta <guidou@chromium.org>
Date: Tue, 3 Oct 2023 20:51:51 +0000
Subject: [PATCH] [MediaDeviceSalt] Enable random salts per storage key

This CL disables the use of a global salt to initialize per-storage-key salts, and uses random salts instead.

This ends the transition period to migrate existing device IDs to the
new storage backend and effectively completes the launch of storage
partitioning for media device salts.

Bug: 1410462,1462956
Change-Id: I8041c10d60e5845ff83a226e8458d14942d82d9c
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4909537
Auto-Submit: Guido Urdaneta <guidou@chromium.org>
Reviewed-by: Elad Alon <eladalon@chromium.org>
Commit-Queue: Elad Alon <eladalon@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1204847}
---
 components/media_device_salt/media_device_salt_service.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/components/media_device_salt/media_device_salt_service.cc b/components/media_device_salt/media_device_salt_service.cc
index e7c1938935051..05298e65bf030 100644
--- a/components/media_device_salt/media_device_salt_service.cc
+++ b/components/media_device_salt/media_device_salt_service.cc
@@ -27,7 +27,7 @@ BASE_FEATURE(kMediaDeviceIdPartitioning,
              base::FEATURE_ENABLED_BY_DEFAULT);
 BASE_FEATURE(kMediaDeviceIdRandomSaltsPerStorageKey,
              "MediaDeviceIdRandomSaltsPerStorageKey",
-             base::FEATURE_DISABLED_BY_DEFAULT);
+             base::FEATURE_ENABLED_BY_DEFAULT);
 
 namespace {
 
